lib/v2/navigation/navigation_service.dart

```diff
@@ -91,6 +92,7 @@ class Routes {
   static final signup = 'signUp';
   static final recoverAccount = 'recoverAccount';
   static final recoverAccountFound = 'recoverAccountFound';
+  static final profile = 'profile';
```

Comment:
Ohh I see the user can move to profile screen from the profile image in wallet, is this for each place where the avatar image appear? if that is the can we can put the navigation inside the avatar widget.

Reply:
Not sure if is for each place.

Reply:
hum, deff from anywhere the appbar shows.